### PR TITLE
Fix small typos, issues, and inconsistencies

### PR DESCRIPTION
**Pull request by andy-sweet**

### CODE REVIEW

File: docs/cryoet_data_portal_docsite_landing.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+The Chan Zuckerberg Institute for Advanced Biological Imaging (CZ Imaging Institute) has developed the prototype CryoET Data Portal providing queryable and organized data from cryoET experiments. The initial target users are segmentation algorithm developers to produce annotations for diverse macromolecules in the tomogram that may be used for high-resolution subtomogram averaging. We currently have 120 tomograms from three datasets contributed by the groups of Julia Mahamid and Jürgen Plitzko. Each of these tomograms have a minimum of “ground truth” point annotations of ribosomes. We also provide a Python API to query and download the data. This portal is not intended to be a final product, but rather, a first step to start a discussion in the wider community as to what is needed with such a portal. We welcome feedback from the community on the data structure, design and functionality. Once we have gathered some feedback we plan to organize a community-wide workshop to discuss next steps.
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:3]: Reflect the correct count of 120 now.

Also, Python is upper case elsewhere which is standard.

File: docs/cryoet_data_portal_docsite_landing.md [hunk 2]
```diff
@@ -42,20 +42,24 @@ Here is a summary of the datasets that have been provided. A full description ca
...
-    `aws s3 ls --no-sign-request s3://cryoet-data-portal-public`
+```
+aws s3 ls --no-sign-request s3://cryoet-data-portal-public
+```
   
 ## Citing the CryoET Data Portal
  
 Data from the portal must acknowledge the data providers and the original publications as follows (inserting of omitting as needed based on which datasets are used):
-Some of the data used in this work was provided by the group (s) of Julia Mahamid (EMBL)/Juergen Plitzko (MPI). The work is described more fully in the publication:
+Some of the data used in this work was provided by the group (s) of Julia Mahamid (EMBL)/Jürgen Plitzko (MPI). The work is described more fully in the publication:
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:58]: Consistency with the umlaut spelling elsewhere on this page

File: docs/cryoet_data_portal_docsite_landing.md [hunk 3]
```diff
@@ -42,20 +42,24 @@ Here is a summary of the datasets that have been provided. A full description ca
...
 
-| Provider | Julia Mahamid | Julia Mahamid | Jürgen Pltzco |
+| Provider | Julia Mahamid | Julia Mahamid | Jürgen Plitzko |
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:60]: Missing i

File: docs/cryoet_data_portal_docsite_landing.md [hunk 4]
```diff
@@ -42,20 +42,24 @@ Here is a summary of the datasets that have been provided. A full description ca
...
 | ----------- | ----------- | ----------- | ----------- |
-| Dataset name | 100000 | 10001 | 10004 |
+| Dataset name | 10000 | 10001 | 10004 |
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:62]: Typo

File: docs/cryoet_data_portal_docsite_landing.md [hunk 5]
```diff
@@ -67,6 +71,6 @@ A description of the CryoET Portal data and its schema is detailed [here](https:
 This is a prototype of a cryoET Data Portal providing queryable and organized data from cryoET experiments. The initial target users are segmentation algorithm developers to produce annotations for diverse macromolecules in the tomogram that may be used for high-resolution subtomogram averaging.  This portal is not intended to be a final product but rather a first step to start a discussion in the wider community as to what is needed with such a portal. We welcome feedback from the community on the data structure, design and functionality. Once we have gathered some feedback we plan to organize a community-wide workshop to discuss next steps.
 
 ## Questions, Feedback, and Issues/Bugs
-- Users are encouraged to submit questions, bugs, and feature requests for the CryoET Data Portal via [Github issues](https://github.com/chanzuckerberg/cryoet-dataportal/issues).
+- Users are encouraged to submit questions, bugs, and feature requests for the CryoET Data Portal via [Github issues](https://github.com/chanzuckerberg/cryoet-data-portal/issues).
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:74]: Broken link due to missing -

File: docs/cryoet_data_portal_docsite_landing.md [hunk 6]
```diff
@@ -42,20 +42,24 @@ Here is a summary of the datasets that have been provided. A full description ca
 ## S3 Bucket Info
 The CryoET Data Portal S3 bucket supports public access. The bucket URL is:
 
-    `s3://cryoet-data-portal-public`
+```
+s3://cryoet-data-portal-public
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:46]: Remove the unnecessary extra backticks

File: docs/cryoet_data_portal_docsite_landing.md [hunk 7]
```diff
@@ -42,20 +42,24 @@ Here is a summary of the datasets that have been provided. A full description ca
...
+```
 
-  To list the bucket contents with the S3 CLI without credentials, please use the following:
+To list the bucket contents with the S3 CLI without credentials, please use the following:
 
-    `aws s3 ls --no-sign-request s3://cryoet-data-portal-public`
+```
+aws s3 ls --no-sign-request s3://cryoet-data-portal-public
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:52]: Remove the unnecessary extra backticks

File: docs/cryoet_data_portal_docsite_landing.md [hunk 8]
```diff
@@ -29,7 +29,7 @@ Here is a summary of the datasets that have been provided. A full description ca
 | Raw movies | Yes | Yes | Yes |
 | Raw tilts | Yes | Yes | No |
 | 3D volumes | Yes | Yes | Yes |
-| Point Annotations (number of particles) | Ribosome, FAS | Ribosome | Ribosome |
+| Point Annotations | Ribosome, FAS | Ribosome | Ribosome |
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:32]: numbers are gone in other columns, so gone here too

File: docs/cryoet_data_portal_docsite_landing.md [hunk 9]
```diff
@@ -29,7 +29,7 @@ Here is a summary of the datasets that have been provided. A full description ca
 | Raw movies | Yes | Yes | Yes |
 | Raw tilts | Yes | Yes | No |
 | 3D volumes | Yes | Yes | Yes |
-| Point Annotations (number of particles) | Ribosome, FAS | Ribosome | Ribosome |
+| Point annotations | Ribosome, FAS | Ribosome | Ribosome |
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:32]: number of particles is not in the other columns, also use lower case a for consistency in the table

File: docs/cryoet_data_portal_docsite_landing.md (hunk 2 above)
Comment (kandarpksk) [reply docs/cryoet_data_portal_docsite_landing.md:58]: Also, in the line just above this, it should be inserting *or* omitting.

**Comment by kandarpksk** [comment]
One other typo I noted:
> A full description can be found below under [Data and schema](https://docs.google.com/document/d/11h0u3YYF1EWCTjxu3ObShx26HgLAfJhn9I_tIaeQ6GI/edit#?usp=sharing).
This is actually a link to a different page rather than a section of the same page.

Suggestion: A full description of the data and schema can be found [here](https://docs.google.com/document/d/11h0u3YYF1EWCTjxu3ObShx26HgLAfJhn9I_tIaeQ6GI/edit#?usp=sharing).

File: docs/cryoet_data_portal_docsite_landing.md [hunk 10]
```diff
@@ -42,20 +42,24 @@ Here is a summary of the datasets that have been provided. A full description ca
...
+To list the bucket contents with the S3 CLI without credentials, please use the following:
 
-    `aws s3 ls --no-sign-request s3://cryoet-data-portal-public`
+```
+aws s3 ls --no-sign-request s3://cryoet-data-portal-public
+```
   
 ## Citing the CryoET Data Portal
  
 Data from the portal must acknowledge the data providers and the original publications as follows (inserting of omitting as needed based on which datasets are used):
```
Comment (andy-sweet) [inline docs/cryoet_data_portal_docsite_landing.md:57]: From @kandarpksk's comment: https://github.com/chanzuckerberg/cryoet-data-portal/pull/10/files#r1224866506

```suggestion
Data from the portal must acknowledge the data providers and the original publications as follows (inserting or omitting as needed based on which datasets are used):
```

File: docs/cryoet_data_portal_docsite_landing.md (hunk 2 above)
Comment (andy-sweet) [reply docs/cryoet_data_portal_docsite_landing.md:58]: > Also, there's a typo in the line just above this: (of -> or)
> 
> > Data from the portal must acknowledge the data providers and the original publications as follows (inserting of omitting as needed based on which datasets are used):
> 
> > Data from the portal must acknowledge the data providers and the original publications as follows (inserting **or** omitting as needed based on which datasets are used):

Fixed.

**Comment by andy-sweet** [comment]
> One other typo I noted:
> 
> > A full description can be found below under [Data and schema](https://docs.google.com/document/d/11h0u3YYF1EWCTjxu3ObShx26HgLAfJhn9I_tIaeQ6GI/edit#?usp=sharing).
> 
> This is actually a link to a different page rather than a section of the same page.
> 
> Suggestion:
> 
> > A full description of the data and schema can be found [here](https://docs.google.com/document/d/11h0u3YYF1EWCTjxu3ObShx26HgLAfJhn9I_tIaeQ6GI/edit#?usp=sharing).

This feels like a bigger change with other questions so probably best outside of this PR.

**Review by EricMaxWang — APPROVED** [review]

**Pull request merged**